arwen/cryptoapi/cryptoei.go

```diff
@@ -317,14 +318,16 @@ func v1_4_verifyEd25519(
 	return 0
 }
 
-//export v1_4_verifySecp256k1
-func v1_4_verifySecp256k1(
+
```

Comment:
do the import for the new function on line 82 like the other imports.

Reply:
done